### PR TITLE
build: add stale configuration

### PR DESCRIPTION
**Pull request by MarshallOfSound**
As in title

Notes: no-notes

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - security
+staleLabel: stale
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+closeComment: >
+  This issue has been closed as it was considered stale, this issue will not be
+  monitored.  If this is a bug and you can reproduce this issue on a supported
+  version of Electron please open a new issue and ensure a repro is provided.
```
Comment (nornagon) [inline .github/stale.yml:14]: let's link to the supported versions doc

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+daysUntilStale: 90
+daysUntilClose: 30
+exemptLabels:
```
Comment (deermichel) [inline .github/stale.yml:3]: enhancement?

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  - discussion
+  - security
+staleLabel: stale
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
```
Comment (nornagon) [inline .github/stale.yml:10]: maybe we should also say something about supported versions here?

File: .github/stale.yml (hunk 1 above)
Comment (nornagon) [inline .github/stale.yml:14]:
```suggestion
  This issue has been closed as it was considered stale, this issue will not be
  monitored.  If this is a bug and you can reproduce this issue on a [supported
  version of Electron](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#timeline) please open a new issue and ensure a repro is provided.
```

File: .github/stale.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+daysUntilClose: 30
+exemptLabels:
+  - discussion
+  - security 🔒
+  - "enhancement :sparkles:"
+staleLabel: stale
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
```
Comment (nornagon) [inline .github/stale.yml:11]: This is what React uses:

> This issue has been automatically marked as stale. **If this issue is still affecting you, please leave any comment** (for example, "bump"), and we'll keep it open. We are sorry that we haven't been able to prioritize it yet. If you have any new additional information, please include it with your comment!

Maybe let's follow their lead and be a bit more gentle, e.g.

```suggestion
  This issue has been automatically marked as stale. **If this issue is still affecting you, please leave any comment** (for example, "bump"), and we'll keep it open. If you have any new additional information—in particular, if this is still reproducible in the [latest version of Electron](https://www.electronjs.org/releases/stable) or in the [beta](https://www.electronjs.org/releases/beta)—please include it with your comment!
```

**Review by VerteDinde — APPROVED** [review]

**Review by deermichel — APPROVED** [review]

File: .github/stale.yml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+daysUntilStale: 90
+daysUntilClose: 30
+exemptLabels:
+  - discussion
+  - security 🔒
+  - "enhancement :sparkles:"
```
Comment (ckerr) [inline .github/stale.yml:6]:
```suggestion
  - "enhancement :sparkles:"
  - "ci-flake ⭕️"
```

File: .github/stale.yml [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+staleLabel: stale
+markComment: >
+  This issue has been automatically marked as stale. **If this issue is still affecting you, please leave any comment** (for example, "bump"), and we'll keep it open. If you have any new additional information—in particular, if this is still reproducible in the [latest version of Electron](https://www.electronjs.org/releases/stable) or in the [beta](https://www.electronjs.org/releases/beta)—please include it with your comment!
+closeComment: >
+  This issue has been closed as it was considered stale, this issue will not be
+  monitored.  If this is a bug and you can reproduce this issue on a [supported
+  version of Electron](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#timeline) please open a new issue and ensure a repro is provided.
```
Comment (ckerr) [inline .github/stale.yml:13]:
```suggestion
  This issue has been closed as it was considered stale and new comments
  will not be monitored.  If this is a bug that you can reproduce it on a [supported
  version of Electron](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#timeline) please open a new issue and ensure a repro is provided.
```

**Comment by nornagon** [comment]
Close 400 GitHub Issues! Developers Hate This One Weird Trick

**Pull request merged**